### ISSUE TITLE
A Weak, Medium, Strong Password Indicator

### ISSUE THREAD

**Issue by this-is-aishwarya** [uday03meh/password-generator#21]
Strongness indicator based on the selected length of the password.

- If the length of the password is 4 <= n <= 6, the password can be indicated weak -> red
- If the length of the password is 7 <= n <= 9, the password can be indicated medium -> orange
- If the length of the password is 10 <= n <= 12, the password can be indicated strong -> green

Let me know if this issue can be assigned.

**Comment by uday03meh** [comment]
Good idea, but which element are you planning to change for indicating these colors?

**Comment by this-is-aishwarya** [comment]
There are two ways I can think of:
1. Introducing a span text that changes on particular slide number -> Text changes to Weak in red, Medium in orange, Strong in green.
2. Changing the slider colour.

**Comment by uday03meh** [comment]
I don't think categorizing passwords as strong. medium and weak makes sense here as all of them are random and not practically possible to crack, still if you want to work on this issue, you can go with the span text idea or can I close this issue?

**Comment by KUNDAN1334** [comment]
i want to contribut in this project, kindly you can assigned it to  me
